scripts/collect_segmentation_intermediates.py

```diff
@@ -67,12 +73,18 @@ def main():
             writer.writerow(segmentation)
 
 
-def read_csv_rows(path, **kwargs):
+def read_csv_as_dicts(path, **kwargs):
     with open(path) as f:
         reader = csv.DictReader(f, **kwargs)
         return [row for row in reader]
 
 
+def read_csv_as_arrays(path, **kwargs):
+    with open(path) as f:
+        reader = csv.reader(f, **kwargs)
+        return [row for row in reader]
```

Comment:
We could read chainsaw with `csv.reader` as well, but I think using headers is more readable in general. In the OOP version, I'd say that the CSV-reading code should be specific to the class, ChainsawParser vs MerizoParser, so we could have the two separate copies of this code.